crates/warpgate-api/src/locator.rs

```diff
@@ -57,6 +57,13 @@ pub struct UrlLocator {
     pub url: String,
 }
 
+/// A OCI Registry locator.
+#[derive(Clone, Debug, Default, Eq, PartialEq)]
+pub struct OciLocator {
```

Comment:
Rename to `RegistryLocator`. OCI is just an implementation detail and not relevant to consumers. I'd wager most people don't even know what it is.